src/status_im/contexts/communities/actions/channel_view_details/view.cljs

```diff
@@ -0,0 +1,108 @@
+          {:keys [description chat-name emoji muted chat-type]
+           :as   chat}                   (rf/sub [:chats/chat-by-id chat-id])
+          pins-count                     (rf/sub [:chats/pin-messages-count chat-id])
+          items                          (rf/sub [:communities/sorted-community-members-section-list
+                                                  community-id])
+          profile-color                  (rf/sub [:profile/customization-color])]
+      [rn/view {:style {:flex 1}}
+       [quo/page-nav
+        {:background :blur
+         :icon-name  :i/arrow-left
+         :on-press   #(rf/dispatch [:navigate-back])
+         :right-side [{:icon-name :i/options
+                       :on-press  #(rf/dispatch [:show-bottom-sheet
+                                                 {:content (fn [] [home.actions/chat-actions
+                                                                   chat
+                                                                   false
+                                                                   true])}])}]}]
+       [quo/text-combinations
+        {:container-style                 style/text-combinations
+         :title                           [quo/channel-name
+                                           {:channel-name chat-name
+                                            :unlocked?    true}]
+         :emoji                           (when (not (string/blank? emoji)) emoji)
+         :emoji-background-color          (colors/resolve-color profile-color theme 10)
+         :title-accessibility-label       :welcome-title
+         :description                     description
+         :description-accessibility-label :welcome-sub-title}]
+       [rn/view {:style style/wrapper}
+        [rn/view
+         {:style style/channel-actions-wrapper}
+         [quo/channel-actions
+          {:actions [{:big?          true
+                      :label         (i18n/label :t/pinned-messages-2)
+                      :color         profile-color
+                      :icon          :i/pin
+                      :counter-value pins-count
+                      :on-press      (fn []
+                                       (rf/dispatch [:dismiss-keyboard])
+                                       (rf/dispatch [:pin-message/show-pins-bottom-sheet
+                                                     chat-id]))}
+                     {:label    (if muted (i18n/label :t/unmute-channel) (i18n/label :t/mute-channel))
+                      :color    profile-color
+                      :icon     (if muted :i/muted :i/activity-center)
+                      :on-press (fn []
+                                  (if muted
+                                    (home.actions/unmute-chat-action chat-id)
+                                    (home.actions/mute-chat-action chat-id chat-type muted)))}]}]]]
+       [members items profile-color]])))
+
+(defn view [] (quo.theme/with-theme view-internal))
```

Comment:
since this is new code it's better to not use the `with-theme` and instead make use of `quo.theme/use-theme-value` hook directly in the view 👍